src/main/drivers/stm32/vcp_hal/usbd_cdc_hid.c

```diff
@@ -235,102 +239,106 @@ uint8_t  *USBD_HID_CDC_GetDeviceQualifierDescriptor (uint16_t *length);       //
-	NULL,
-	NULL,
-	NULL,
-	USBD_HID_CDC_GetFSCfgDesc,
-	NULL,
-	USBD_HID_CDC_GetDeviceQualifierDescriptor,
+    USBD_HID_CDC_Init,
+    USBD_HID_CDC_DeInit,
+    USBD_HID_CDC_Setup,
+    NULL,                 /* EP0_TxSent, */
+    USBD_HID_CDC_EP0_RxReady,
+    USBD_HID_CDC_DataIn,
+    USBD_HID_CDC_DataOut,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    USBD_HID_CDC_GetFSCfgDesc,
+    NULL,
+    USBD_HID_CDC_GetDeviceQualifierDescriptor,
 };
 
 static uint8_t USBD_HID_CDC_Init(USBD_HandleTypeDef *pdev, uint8_t cfgidx)
 {
-	//Init CDC
-	USBD_CDC.Init(pdev, cfgidx);
+    //Init CDC
+    USBD_CDC.Init(pdev, cfgidx);
 
-	//Init HID
-	USBD_HID.Init(pdev, cfgidx);
+    //Init HID
+    USBD_HID.Init(pdev, cfgidx);
 
-	return USBD_OK;
+    return USBD_OK;
 }
 
 static uint8_t USBD_HID_CDC_DeInit(USBD_HandleTypeDef *pdev, uint8_t cfgidx)
 {
-	//DeInit CDC
-	USBD_CDC.DeInit(pdev, cfgidx);
+    //DeInit CDC
+    USBD_CDC.DeInit(pdev, cfgidx);
 
-	//DeInit HID
-	USBD_HID.DeInit(pdev, cfgidx);
+    //DeInit HID
+    USBD_HID.DeInit(pdev, cfgidx);
 
-	return USBD_OK;
+    return USBD_OK;
 }
 
 static uint8_t USBD_HID_CDC_Setup(USBD_HandleTypeDef *pdev, USBD_SetupReqTypedef *req)
 {
-	switch(req->bmRequest & USB_REQ_RECIPIENT_MASK) {
-	case USB_REQ_RECIPIENT_INTERFACE:
-		if (req->wIndex == HID_INTERFACE) {
-			return USBD_HID.Setup(pdev, req);
-		}
-		else {
-			return USBD_CDC.Setup(pdev, req);
-		}
-		break;
-	case USB_REQ_RECIPIENT_ENDPOINT:
-		if (req->wIndex == HID_EPIN_ADDR) {
-			return USBD_HID.Setup(pdev, req);
-		} else {
-			return USBD_CDC.Setup(pdev, req);
-		}
-		break;
-	}
-
-	return USBD_OK;
+    switch(req->bmRequest & USB_REQ_RECIPIENT_MASK) {
+        case USB_REQ_RECIPIENT_INTERFACE:
+            if (req->wIndex == HID_INTERFACE) {
+                return USBD_HID.Setup(pdev, req);
+            } else {
+                return USBD_CDC.Setup(pdev, req);
+            }
+            break;
+        case USB_REQ_RECIPIENT_ENDPOINT:
+            if (req->wIndex == HID_EPIN_ADDR) {
+                return USBD_HID.Setup(pdev, req);
+            } else {
+                return USBD_CDC.Setup(pdev, req);
+            }
+            break;
+      }
+
+    return USBD_OK;
 }
 
 static uint8_t USBD_HID_CDC_EP0_RxReady  (USBD_HandleTypeDef *pdev)
 {
-	return (USBD_CDC.EP0_RxReady(pdev));
+    return (USBD_CDC.EP0_RxReady(pdev));
 }
 
 static uint8_t USBD_HID_CDC_DataIn(USBD_HandleTypeDef *pdev, uint8_t epnum)
 {
-	if (epnum == (CDC_IN_EP &~ 0x80)) {
-		return USBD_CDC.DataIn(pdev, epnum);
-	}
-	else {
-		return USBD_HID.DataIn(pdev, epnum);
-	}
-
-	return USBD_OK;
+    if (epnum == (CDC_IN_EP &~ 0x80)) {
+        return USBD_CDC.DataIn(pdev, epnum);
+    } else {
+        return USBD_HID.DataIn(pdev, epnum);
+    }
+
+    return USBD_OK;
 }
 
 static uint8_t USBD_HID_CDC_DataOut(USBD_HandleTypeDef *pdev, uint8_t epnum)
 {
-	if (epnum == (CDC_OUT_EP &~ 0x80)) {
-		return (USBD_CDC.DataOut(pdev, epnum));
-	}
+  if (epnum == (CDC_OUT_EP &~ 0x80)) {
```

Comment:
4 spaces or 2 - don't know if this in coding style anyway :)

Reply:
these are old files... I cleaned up most of it will clean up these now